test/compilable/extra-files/vcg-ast.d.cg

```diff
@@ -129,11 +139,6 @@ RTInfo!(C)
 {
 	enum immutable(void)* RTInfo = null;
 
-}
-NoPointersBitmapPayload!1$?:32=u|64=LU$
-{
-	enum $?:32=uint|64=ulong$[1] NoPointersBitmapPayload = 0$?:32=u|64=LU$;
```

Comment:
Since semantic3 is made eagerly on template instances coming from `typeof` expressions, this is now known earlier, however, I don't really get the changes.